stories/object-number/object-number.stories.js

```diff
@@ -43,63 +41,69 @@ Make sure that the object number is properly described and semantically understa
-<span class="fd-object-number fd-object-number--negative">
-    <span class="fd-object-number__text">-2,000.00</span><span class="fd-object-number__unit">EUR</span>
-</span>
-<span class="fd-object-number fd-object-number--critical">
-    <span class="fd-object-number__text">0.00</span><span class="fd-object-number__unit">EUR</span>
-</span>
-<span class="fd-object-number fd-object-number--positive">
-    <span class="fd-object-number__text">1,000.00</span><span class="fd-object-number__unit">EUR</span>
-</span>
-<span class="fd-object-number fd-object-number--informative">
-    <span class="fd-object-number__text">100.00</span><span class="fd-object-number__unit">EUR</span>
-</span>
-<span class="fd-object-number">
-    <span class="fd-object-number__text">999.00</span><span class="fd-object-number__unit">EUR</span>
-</span>
+<div class="fddocs-container">
+    <span class="fd-object-number fd-object-number--negative">
+        <span class="fd-object-number__text">-2,000.00</span><span class="fd-object-number__unit">EUR</span>
+    </span>
+    <span class="fd-object-number fd-object-number--critical">
+        <span class="fd-object-number__text">0.00</span><span class="fd-object-number__unit">EUR</span>
+    </span>
+    <span class="fd-object-number fd-object-number--positive">
+        <span class="fd-object-number__text">1,000.00</span><span class="fd-object-number__unit">EUR</span>
+    </span>
+    <span class="fd-object-number fd-object-number--informative">
+        <span class="fd-object-number__text">100.00</span><span class="fd-object-number__unit">EUR</span>
+    </span>
+    <span class="fd-object-number">
+        <span class="fd-object-number__text">999.00</span><span class="fd-object-number__unit">EUR</span>
+    </span>
+</div>
 `;
 
 /**
  * In some cases, it can be useful to enlarge the text for the object number. To display the object number in large text, add the `--large` modifier class to the element.
  */
 
 export const large = () => `
-<span class="fd-object-number fd-object-number--large fd-object-number--negative">
-    <span class="fd-object-number__text">-2,000.00</span><span class="fd-object-number__unit">EUR</span>
-</span>
-<span class="fd-object-number fd-object-number--large fd-object-number--critical">
-    <span class="fd-object-number__text">0.00</span><span class="fd-object-number__unit">EUR</span>
-</span>
-<span class="fd-object-number fd-object-number--large fd-object-number--positive">
-    <span class="fd-object-number__text">1,000.00</span><span class="fd-object-number__unit">EUR</span>
-</span>
-<span class="fd-object-number fd-object-number--large fd-object-number--informative">
-    <span class="fd-object-number__text">100.00</span><span class="fd-object-number__unit">EUR</span>
-</span>
-<span class="fd-object-number fd-object-number--large">
-    <span class="fd-object-number__text">999.00</span><span class="fd-object-number__unit">EUR</span>
-</span>
+<div class="fddocs-container">
```

Comment:
Most of these diffs are due to adding `<div class="fddocs-container">` because I got rid of the docs override styling for these components.